deps/node/deps/uv/include/uv.h

```diff
@@ -1763,6 +1763,7 @@ UV_EXTERN int uv_thread_create_ex(uv_thread_t* tid,
                                   void* arg);
 UV_EXTERN uv_thread_t uv_thread_self(void);
 UV_EXTERN int uv_thread_join(uv_thread_t *tid);
+UV_EXTERN int uv_thread_setname(const char* name);
```

Comment:
```suggestion
// @lwnode
UV_EXTERN int uv_thread_setname(const char* name);
```

Does this API need the `// @lwnode` tag like `uv_watcher_queue_empty`?